src/sentry/replays/lib/new_query/parsers.py

```diff
@@ -22,6 +22,21 @@ def parse_int(value: str) -> int:
     return int(parse_float(value))
 
 
+def parse_duration(value: str) -> int:
+    """
+    Assert that second resolution is given. The input and output of this fx is still in milliseconds, to match the
+    output of api.event_search.parse_search_query
+    """
+    milliseconds = parse_int(value)
+    if milliseconds % 1000:
+        # TODO: remove once we support milliseconds.
+        # TODO: this error isn't actually returned to the frontend, it's caught and then we raise a ParseError
```

Comment:
2nd todo is for me to finish before merging

Reply:
Actually I won't touch error msgs returned to the user in this PR, can do in a follow-up.
Proposed change: include the message of `CouldNotParseValue` in this handler in [__init__.py](https://github.com/getsentry/sentry/blob/811c28893c0ad0eed86a80b27bcb8f2578c50b17/src/sentry/replays/usecases/query/__init__.py#L113-L113)